### PR TITLE
Merge bubble-sort reference into master

### PR DESCRIPTION
**Pull request by typekev**

### CODE REVIEW

File: bubble-sort/index.js [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+        console.log("after: ", arr.join(" "));
+      }
+    }
+    /**
+     * The condition on this do...while loop is that is must run at least once and until swapped is false.
+     * See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/do...while#Syntax
+     */
+  } while (swapped);
+  /**
+   * We exit our do...while loop, at this point the bubble sort algorithm has completed it's course.
```
Comment (fredsh) [inline bubble-sort/index.js]: typo here. should be "its course"

**Review by fredsh — COMMENTED** [review]
I think it would be good to introduce control structure separatelz from sort algorithm so that the example can be more focused.

File: bubble-sort/index.js (hunk 1 above)
Comment (typekev) [reply bubble-sort/index.js]: Thanks, good catch!

**Comment by typekev** [comment]
> I think it would be good to introduce control structure separately from sort algorithm so that the example can be more focused.

When we have the appropriate branch I will definitely consider it. Maybe we can open an issue for this, so that it can be tracked.

**Pull request merged**